packages/manager/apps/web/client/app/hosting/database/private-sql-activation/private-sql-activation.html

```diff
@@ -0,0 +1,73 @@
+<oui-back-button data-on-click="$ctrl.goToHosting()">
+    <span data-translate="privatesql_activation_back"></span>
+</oui-back-button>
+<oui-stepper>
+    <oui-step-form data-header="{{:: 'privatesql_activation_step_1' | translate }}"
+                   data-valid="$ctrl.hosting && $ctrl.option && $ctrl.version && !$ctrl.error.hosting"
+                   data-prevent-next="true"
+                   data-loading="$ctrl.loading.hosting"
+                   data-editable="!$ctrl.loading.checkout"
+                   data-on-submit="$ctrl.fetchCheckoutInformations()">
+        <oui-message data-type="error"
```

Comment:
For discussion : Maybe these `oui-message` should be handled inside the `oui-step-form` component

Reply:
yes it should be good but not really the scope of this PR